packages/coinjoin/src/constants.ts

```diff
@@ -24,6 +24,7 @@ export const ROUND_SELECTION_REGISTRATION_OFFSET = 30000;
 
 // fallback values for status request
 // usage of these values is extremely unlikely, there would have to be a change in the coordinator's API
-export const MAX_COORDINATOR_FEE_RATE = 0.003;
+export const PLEBS_DONT_PAY_THRESHOLD = 1000000;
```

Comment:
what a name

Reply:
Not my invention :)

Reply:
I know, I simply wanted to recognise how great it is :D